Build/QIR_validation.ps1

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+$ErrorActionPreference = 'Stop'
+
+& "$PSScriptRoot/set-env.ps1"
+$all_ok = $True
+
+function Build-One {
```

Comment:
I'd suggest to take both the project as well as the command line arguments here for execution.

Reply:
I'm looking for fairly detailed help with this, as I am not very skilled with powershell.

Reply:
done